staging/src/k8s.io/apimachinery/pkg/util/cache/expiring.go

```diff
@@ -0,0 +1,121 @@
+package cache
+
+import (
+	"sync"
+	"time"
+)
+
+type ExpiringDeleter interface {
```

Comment:
Is this just for testing? If so make private?

Reply:
Fixed.